seed_scheduler/settings.py

```diff
@@ -188,6 +188,9 @@
     'seed_scheduler.scheduler.tasks.queue_tasks': {
         'queue': 'priority',
     },
+    'seed_scheduler.scheduler.tasks.requeue_failed_tasks': {
+        'queue': 'priority',
+    },
```

Comment:
The thing about the re-queue task is that it is one task that will re-queue all the currently failed tasks in the DB. This is just the kickoff task, the actual re-delivery of the tasks will still happen via the existing `deliver_task()` task and hence will be on lowpriority queue as well.